WooCommerce/src/main/kotlin/com/woocommerce/android/ui/products/addons/order/OrderedAddonViewModel.kt

```diff
@@ -0,0 +1,105 @@
+package com.woocommerce.android.ui.products.addons.order
+
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.SavedStateHandle
+import com.woocommerce.android.extensions.unwrap
+import com.woocommerce.android.model.Order
+import com.woocommerce.android.model.ProductAddon
+import com.woocommerce.android.model.ProductAddonOption
+import com.woocommerce.android.ui.products.addons.AddonRepository
+import com.woocommerce.android.util.CoroutineDispatchers
+import com.woocommerce.android.viewmodel.ScopedViewModel
+import dagger.hilt.android.lifecycle.HiltViewModel
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.withContext
+import javax.inject.Inject
+
+@HiltViewModel
+class OrderedAddonViewModel @Inject constructor(
+    savedState: SavedStateHandle,
+    private val dispatchers: CoroutineDispatchers,
+    private val addonsRepository: AddonRepository
+) : ScopedViewModel(savedState) {
+    companion object {
+        private const val addonAttributeGroupSize = 3
+    }
+
+    private val _orderedAddons = MutableLiveData<List<ProductAddon>>()
+    val orderedAddonsData = _orderedAddons
```

Comment:
🔧 This is exposed as `MutableLiveData` - I think we want here to expose not mutable variant, is that correct?
```suggestion
    val orderedAddonsData: LiveData<List<ProductAddon>> = _orderedAddons
```

Reply:
Fixed! And also fixed the internal `value` access to the LiveData inside the ViewModel: https://github.com/woocommerce/woocommerce-android/pull/4590/commits/3d2ea470c055d5ed560bfdfd541cf333d02fde53